litestar/typing.py

```diff
@@ -225,10 +219,12 @@ def __hash__(self) -> int:
     def _extract_metadata(
         cls, annotation: Any, name: str | None, default: Any, metadata: tuple[Any, ...], extra: dict[str, Any] | None
     ) -> tuple[KwargDefinition | None, dict[str, Any]]:
+        from litestar.contrib.pydantic.utils import is_pydantic_constrained_field, is_pydantic_field_info
```

Comment:
Same doubt as mentioned before. I think this would cause the `MissingDepdendencyError` to be raised in the case of `pydantic` not being installed.